src/components/artistas/ArtistaForm.jsx

```diff
@@ -245,6 +250,58 @@ class ArtistaForm extends Component {
     }
   }
 
+  handleVimeoChange(id) {
+
+    if(id !== undefined && id !== null && id !== '') { // If id has a value
+
+      // Call to get the video data
+      const videoData = getVimeoData(id, response => {
```

Comment:
are you just calling this to a `const` so you can call the function? linter doesn't like it but not sure it matters too much